lms/static/js/student_account/views/FinishAuthView.js

```diff
@@ -146,6 +148,8 @@
                     */
                     this.updateTaskDescription(gettext("Adding the selected course to your cart"));
                     shoppingCartInterface.addCourseToCart( this.courseId );
+                } else if ( this.enrollmentAction === 'add_to_ecomm_cart' && this.courseId) {
```

Comment:
ASIDE: The more of these we add the more this section looks like it could be a little cleaner with a switch statement on `enrollmentAction`